### DISCUSSION TITLE
When select usd prim in VP2.0,the attribute editor not refresh

### DISCUSSION THREAD

**Discussion by ActiveXSturcture** [Autodesk/maya-usd#1680] [Q&A] [answered]
I import USD proxy using  Create -> Universal Scene Description -> stage from file;
But when i select proxy mesh in viewport i found i can`t see any USD attribute in attribute editor;
I`m using Maya 2020.4,download from release package v0.12.0 and test in Centos7,and enable show array attribute toggle;Is there someone can tell me which step i`m wrong?

**Comment by seando-adsk** [comment, marked as answer]
Displaying of USD prims in the Attribute Editor requires UFE v2 which unfortunately Maya 2020 does not have. UFE v2 (and AE display of USD) is only available in Maya 2022 and later.

You have actually discovered a small problem - that `Show Array Attributes` menu item should not appear in Maya 2020. @santosg87 Can you make sure to log this internally.

Sean

**Reply by ActiveXSturcture** [reply]
Thank you for your answer,I just wondering is there are any feature board for different Maya version,I just realize maya 2020 usd plugin can not add any prim in stage which i create using stage with new layer.Did maya 2020 usd plugin only support USD file convert and showing usd content in viewport?

**Comment by kxl-adsk** [comment]
MayaUSD for Maya 2020 has pretty limited support for direct workflows. Besides already mentioned data model translation and visualization, UFE v1 also enabled outliner support and viewport manipulation for CommonAPI xform stacks.

Cheers,
KxL
